misc/null-3k/main.tf

```diff
@@ -0,0 +1,3 @@
+resource "null_resource" "null" {
+    count = 3000
```

Comment:
would be nice to have this as a var with 3000 as a default, just so we can play with it without changing the template.